root/static/scripts/release-editor/utils.js

```diff
@@ -137,6 +137,10 @@ utils.cleanWebServiceData = function (data) {
         clean.comment = data.disambiguation;
     }
 
+    if (data["isrcs"]) {
+        clean.isrcs = _.map(data["isrcs"], cleanIsrc);
```

Comment:
BTW, you can use `data.isrcs` instead of `data["isrcs"]` here, the brackets/quotes are only needed on sort-name/artist-credit 'cause of the hyphens.